demisto_sdk/commands/init/initiator.py

```diff
@@ -99,6 +108,74 @@ def init(self):
             self.get_created_dir_name(created_object="pack")
             return self.pack_init()
 
+    def convert_contribution_to_pack(self):
+        '''Create a Pack in the content repo from the contents of a contribution zipfile'''
+        packs_dir = os.path.join(get_content_path(), 'Packs')
+        metadata_dict = {}
+        with zipfile.ZipFile(self.contribution) as zipped_contrib:
+            with zipped_contrib.open('metadata.json') as metadata_file:
+                print_color(f'Pulling relevant information from {metadata_file.name}', LOG_COLORS.NATIVE)
+                metadata = json.loads(metadata_file.read())
+                pack_name = metadata.get('name', 'ContributionPack')
+                metadata_dict['name'] = pack_name
+                metadata_dict['author'] = metadata.get('author', '')
+                metadata_dict['support'] = metadata.get('support', '')
+                metadata_dict['url'] = metadata.get('supportDetails', {}).get('url', '')
+                metadata_dict['email'] = metadata.get('supportDetails', {}).get('email', '')
+        if os.path.exists(os.path.join(packs_dir, pack_name)):
+            print_color(
+                f'Modifying pack name because pack {pack_name} already exists in the content repo', LOG_COLORS.NATIVE
+            )
+            if pack_name[-2].lower() == 'v' and pack_name[-1].isdigit():
+                # increment by one
+                pack_name = pack_name[:-1] + str(int(pack_name[-1]) + 1)
+            else:
+                pack_name += 'V2'
+            print_color(f'New pack name is "{pack_name}"', LOG_COLORS.NATIVE)
+        pack_dir = os.path.join(packs_dir, pack_name)
+        os.mkdir(pack_dir)
+        shutil.unpack_archive(filename=self.contribution, extract_dir=pack_dir)
+        pack_subdirectories = get_child_directories(pack_dir)
+        for pack_subdir in pack_subdirectories:
+            basename = os.path.basename(pack_subdir)
+            if basename in ENTITY_TYPE_TO_DIR:
+                dst_name = ENTITY_TYPE_TO_DIR.get(basename)
+                src_path = os.path.join(pack_dir, basename)
+                dst_path = os.path.join(pack_dir, dst_name)
+                content_item_dir = shutil.move(src_path, dst_path)
+                if basename in {'integration', 'automation'}:
```

Comment:
lets use the constants we have for these entities
lets also search for `script` in case we add it in the future